doc/encoders.texi

```diff
@@ -3991,6 +3991,20 @@ Average variable bitrate.
 Each encoder also has its own specific options:
 @table @option
 
+@item av1_vaapi
+@option{profile} sets the value of @emph{seq_profile}.
+@option{tier} sets the value of @emph{seq_tier}.
+@option{level} sets the value of @emph{seq_level_idx}.
+
+@table @option
+@item tile_cols
+Set tile columns number (default is 1):
+@item tile_rols
+Set tile rows number (default is 1):
```

Comment:
Please add tiles instead of tiles_cols / tiles_rols.
-tiles             <image_size> E..V....... Tile columns x rows